src/main/java/org/prebid/server/bidder/openx/proto/ExtImpOpenx.java

```diff
@@ -0,0 +1,25 @@
+package org.prebid.server.bidder.openx.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.Map;
+
+/**
+ * Defines the contract for bidrequest.imp[i].ext.openx
+ */
+@Builder
```

Comment:
Should we use @AllArgsConstructor when class has 4 or less fields? Actually I am not sure what should we use for exactly 4 fields

Reply:
I thought that 3 is the constraint...